app/helpers/application_helper.rb

```diff
@@ -124,4 +124,8 @@ def storage_location_for_source(source_object)
     end
     current_organization.default_storage_location
   end
+
+  def set_default_location(source_object)
```

Comment:
Why is this called "set\_" when it's not setting anything? Maybe change this to just `default_location`?